doc/Components.md

```diff
@@ -0,0 +1,164 @@
+# Components of CORTX
+
+## A list of the Components of CORTX
+
+1. [Motr](https://github.com/seagate/cortx-motr)
+1. [Provisioner](https://github.com/Seagate/cortx-prvsnr)
+1. [Experiments](https://github.com/Seagate/cortx-experiments)
+1. [HA](https://github.com/Seagate/cortx-ha/)
+1. [HARE](https://github.com/Seagate/cortx-hare/)
+1. [Management Portal](https://github.com/Seagate/cortx-management-portal)
+1. [Manager](https://github.com/Seagate/cortx-manager)
+1. [Monitor](https://github.com/Seagate/cortx-monitor)
+1. [POSIX](https://github.com/Seagate/cortx-posix)
+1. [S3-Server](https://github.com/Seagate/cortx-s3server)
+1. [Utils](https://github.com/Seagate/cortx-utils)
+
+## Definitions and Links to each of the components
+
+### Motr
+
+At the core of CORTX lies Motr. Motr is a distributed object storage system, targeting mass capacity storage configurations. 
+To ensure the most efficient storage utilization, Motr interacts directly with block devices (i.e., it does not _layer_ on a local file system).
+The Motr design was heavily influenced by the Lustre file system, NFSv4 and database technology. It must be noted that traditional file system properties 
+(hierarchical directory namespace, strong POSIX consistency guarantees, &c.) are no longer desirable or achievable at mass capacity. Instead, Motr is a more
+general storage system that provides an optional file system interface. This allows wider range of deployments, including cloud.
+
+Following are the features of CORTX Motr: 
```

Comment:
I think you should remove this bit about features.  Too much info for this file.